controller_manager/src/controller_manager.cpp

```diff
@@ -188,6 +201,26 @@ ControllerManager::ControllerManager(
   init_services();
 }
 
+void ControllerManager::init_resource_manager_cb(const std_msgs::msg::String & robot_description)
+{
+  RCLCPP_ERROR(
+    get_logger(), "'init_resource_manager_cb called with %s", robot_description.data.c_str());
```

Comment:
Making this output nice.
```suggestion
  RCLCPP_INFO("Received robot description from topic.");
  RCLCPP_DEBUG(
    get_logger(), "'Received robot description: \n%s", robot_description.data.c_str());
```

Reply:
Done.